streamer/src/nonblocking/quic.rs

```diff
@@ -647,6 +653,12 @@ async fn packet_batch_sender(
                 || (!packet_batch.is_empty() && elapsed >= coalesce)
             {
                 let len = packet_batch.len();
+                track_streamer_fetch_packet_performance(
```

Comment:
we should track how long this extra loop is taking

Reply:
done.